rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_15.json

```diff
@@ -0,0 +1,646 @@
+                                                            "specification_method": "SIMPLE",
+                                                            "design_electric_power": 100
+                                                        }
+                                                    ],
+                                                    "return_fans": [
+                                                        {
+                                                            "id": "Return Fan 1",
+                                                            "specification_method": "SIMPLE",
+                                                            "design_electric_power": 100
+                                                        }
+                                                    ],
+                                                    "minimum_outdoor_airflow": 153.38291903999993
+                                                }
+                                            }
+                                        ]
+                                    }
+                                ]
+                            }
+                        ]
+                    }
+                ]
+            },
+            "baseline": {
+                "id": "ASHRAE229 1",
+                "ruleset_model_descriptions": [
+                    {
+                        "id": "RMD 1",
+                        "buildings": [
+                            {
+                                "id": "Building 1",
+                                "building_open_schedule": "Required Building Schedule 1",
+                                "building_segments": [
+                                    {
+                                        "id": "Building Segment 1",
+                                        "zones": [
+                                            {
+                                                "id": "Thermal Zone 1",
+                                                "thermostat_cooling_setpoint_schedule": "Required Cooling Schedule 1",
+                                                "thermostat_heating_setpoint_schedule": "Required Heating Schedule 1",
+                                                "terminals": [
+                                                    {
+                                                        "id": "Air Terminal",
+                                                        "is_supply_ducted": true,
+                                                        "type": "CONSTANT_AIR_VOLUME",
+                                                        "served_by_heating_ventilating_air_conditioning_system": "System 9",
+                                                        "primary_airflow": 471.94744319999984,
+                                                        "supply_design_heating_setpoint_temperature": 40.5555555555556
```

Comment:
If the ```supply_design_heating_setpoint_temperature``` temp is 105 degF, the ```design_airflow``` in supply fans has to be greater than the ```minimum_outdoor_airflow``` value. However, this case was already created in (a), you may wat to set the ```supply_design_heating_setpoint_temperature``` not equal to 105 degF, and ```design_airflow``` =```minimum_outdoor_airflow```

Reply:
Adjusted the `supply_heating_setpoint_temperature` so the code can reach testing for Case 2.

Reply:
Hi @JacksonJ-KC ,

I tested your new test JSON on 19-15 and had to make a few updates to our python code, but with this fix everything appears to pass/fail/etc. as expected.

Reply:
Excellent news!

Reply:
Thank you both! I also verified that all the cases passed as expected.